tools/fuzzer/platform/imx8-host.c

```diff
@@ -0,0 +1,515 @@
+// SPDX-License-Identifier: BSD-3-Clause
+//
+// Copyright(c) 2019 Intel Corporation. All rights reserved.
+// Copyright 2020 NXP
+//
+// Author: Diana Cretu <diana.cretu@nxp.com>
+
+/* Core IA host MU support for i.MX8 audio DSP. */
+
+#include <errno.h>
```

Comment:
Remove 'Core IA' I think this is related to Intel. Just keep 'Host MU support for i.MX8 audio DSP'